rocksdb_admin/admin_handler.h

```diff
@@ -126,6 +126,9 @@ class AdminHandler : virtual public AdminSvIf {
   // Dump stats for all DBs as a text string
   std::string DumpDBStatsAsText() const;
 
+  // Get all the db names held by the AdminHandler
+  const std::vector<std::string> getAllDBNames();
```

Comment:
remove const.
https://stackoverflow.com/questions/8716330/purpose-of-returning-by-const-value